docs/topics/gradle/gradle-compiler-options.md

```diff
@@ -116,13 +151,21 @@ tasks.named('compileKotlin', KotlinCompilationTask) {
 
 Here is a complete list of options for Gradle tasks:
 
+### Common attributes
+
+| Name              | Description                                     | Possible values           | Default value |
+|-------------------|-------------------------------------------------|---------------------------|---------------|
+| `optIn`           | Property to configure opt-in compiler arguments | `listOf( /* opt-ins */ )` | `emptyList()` |
+| `progressiveMode` | Enable progressive compiler mode                | `true`, `false`           | `false`       |
```

Comment:
Maybe we should provide links here to our extended documentation for `optIn` and `progressiveMode` ?